modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -18,9 +18,17 @@
 # [*size_critical*]
 #   The number of messages on a queue to trigger a critical alert.
 #
+# [*latency_warning*]
+#   The warning threshold for a sidekiq queue in minutes.
+#
+# [*latency_critical*]
+#   The critical threshold for a sidekiq queue in minutes.
+#
 define govuk::apps::email_alert_api::sidekiq_queue_check(
   $size_warning,
   $size_critical,
```

Comment:
```puppet
  $size_warning = '75000'
  $size_critical = '100000',
```

I wonder if it's worth doing this since they seem to be the same across all the queues.

Reply:
While that would save 4 lines of config, it's probably more advantageous to use the extra lines and have all the configuration in one place.

Reply:
Sure, sounds reasonable. I wonder if we should at least pull them out into variables to avoid them getting out of sync between queues, but that's probably not a big deal.